packages/graphql/src/interfaces/graphqlServiceOption.interface.ts

```diff
@@ -45,5 +45,5 @@ export interface GraphqlServiceOption extends BackendServiceOption {
    * When false, searchTerms may be manipulated to be functional with certain filters eg: string only filters.
    * When true, JSON.stringify is used on the searchTerms and used in the query "as-is". It is then the responsibility of the developer to sanitise the `searchTerms` property if necessary.
    */
-  verbatimSearchTerms?: boolean;
+  useVerbatimSearchTerms?: boolean;
```

Comment:
this one is ok as `useVerbatimSearchTerms` because it's a flag, so no change required here